service/cloudFormation/sam-template-empty.yml

```diff
@@ -18,7 +18,7 @@ Resources:
       Description: Lambda Function to run playwright test
 #add name of the text
       FunctionName: string
-      MemorySize: 512
+      MemorySize: 1024
```

Comment:
Do you think maybe we should make it configurable from the user's side, with a default value of 1024? The same goes for the timeout parameter.